### ISSUE TITLE
Fail to render BatchNormalization layer

### ISSUE THREAD

**Issue by walkerwzy** [lutzroeder/netron#770]
- Netron app and version:  5.0.1
- OS and browser version:  macOS Big Sur(11.5)

Steps to Reproduce:

1. write a `Conv` layer with bias disabled
2. followed by a `BatchNormalization` layer

you'll found the `bias` still shows in the `Conv2d` node, and there's no `BatchNormalization` node

demo code in pytorch
```
class myModel(nn.Module):
    def __init__(self):
        super().__init__()
        self.layers = nn.Sequential(
            nn.Conv2d(3, 20, 3, stride=2, bias=False),
            nn.Conv2d(20, 40, 3, stride=2, bias=False),
            nn.BatchNorm2d(40),
            nn.ReLU(inplace=True),
            nn.Flatten(),
            nn.Linear(1000, 8)
        )

    def forward(self, x):
        return self.layers(x)

m = myModel()
torch.onnx.export(m, (torch.ones(1,3,24,24),), 'test.onnx')
```
![88151626341848_ pic_hd](https://user-images.githubusercontent.com/1918101/125768310-d34d5d23-d525-4b45-a985-231ffb7ec5f2.jpg)

the exported onnx file: [test.onnx.zip](https://github.com/lutzroeder/netron/files/6822087/test.onnx.zip)

**Comment by lutzroeder** [comment]
This is a question for PyTorch or ONNX exporter.
